Makefile

```diff
@@ -29,6 +29,18 @@ test:
 	docker-compose up -d db queue
 	docker-compose -f docker-compose.test.yml up --build --force-recreate --exit-code-from unit_test unit_test
 
+postman:
+	# This target should be executed by passing in an argument for the environment (dev/test/sbx)
+	# and if needed a token.
+	# For example: make postman env=test token=<MY_TOKEN>
+	docker-compose -f docker-compose.test.yml run postman_test test/$(env).postman_environment.json --global-var "token=$(token)"
+
+
+postman-local:
+	docker-compose up -d
+	sleep 30
+	docker-compose -f docker-compose.test.yml run postman_test test/local.postman_environment.json
```

Comment:
We are going to need to capture the exit code from the container similar to the other make targets. Otherwise the exit code won’t propagate to Travis properly.

Reply:
It looks like run actually just returns the exit code by default with no additional specifications required.

Reply:
confirmed on this build  https://travis-ci.org/CMSgov/bcda-app/builds/473104369?utm_source=github_status&utm_medium=notification